step.rb

```diff
@@ -5,90 +5,6 @@
 
 build_slug = ENV['build_slug']
 
```

Comment:
```
get_ios_license = ENV['xamarin_ios_license'].eql?("yes") ? true : false
get_android_license = ENV['xamarin_android_license'].eql?("yes") ? true : false
get_mac_license = ENV['xamarin_mac_license'].eql?("yes") ? true : false
```

These inputs are not used in the code, please remove them from the step.yml as well.